plugins/performance-lab/includes/admin/load.php

```diff
@@ -213,6 +213,46 @@ function perflab_dismiss_wp_pointer_wrapper(): void {
 }
 add_action( 'wp_ajax_dismiss-wp-pointer', 'perflab_dismiss_wp_pointer_wrapper', 0 );
 
+/**
+ * Gets the path to a script or stylesheet.
+ *
+ * @since n.e.x.t
+ *
+ * @param string      $src_path Source path.
+ * @param string|null $min_path Minified path. If not supplied, then '.min' is injected before the file extension in the source path.
+ * @return string URL to script or stylesheet.
+ */
+function perflab_get_asset_path( string $src_path, ?string $min_path = null ): string {
+	if ( null === $min_path ) {
+		// Note: wp_scripts_get_suffix() is not used here because we need access to both the source and minified paths.
+		$min_path = (string) preg_replace( '/(?=\.\w+$)/', '.min', $src_path );
+	}
+
+	$force_src = false;
+	if (
+		( WP_DEBUG || wp_get_environment_type() === 'local' )
+		&&
+		! file_exists( PERFLAB_PLUGIN_DIR_PATH . $min_path )
```

Comment:
So this means a regular (non `WP_DEBUG`) site would not have the presence of the file checked right? That makes sense to me because in the built plugin it would always exist, and it avoids a `file_exists()` check we don't need in that situation.

That said, I don't think we should use `wp_get_environment_type()` here. `WP_DEBUG` alone should be sufficient as a differentiator. We don't know whether a local environment is used to work on this plugin or just a general site where the built version of this plugin is installed - in the latter case there's no difference from a production environment for what we're doing here.